enterprise/backup/src/main/java/org/neo4j/backup/BackupProtocolService.java

```diff
@@ -93,14 +93,12 @@
             " perform a full backup at this point. You can modify this time interval by setting the '" +
             GraphDatabaseSettings.keep_logical_logs.name() + "' configuration on the database to a higher value.";
 
-    static final String DIFFERENT_STORE_MESSAGE = "Target directory contains full backup of a logically different store.";
-
     private final Supplier<FileSystemAbstraction> fileSystemSupplier;
     private final LogProvider logProvider;
     private final Log log;
     private final OutputStream logDestination;
     private final Monitors monitors;
-    private final Optional<PageCache> pageCacheOptional;
+    private final PageCache pageCache;
```

Comment:
Does pageCache need to be closed?

Reply:
The page cache isn't opened here, so closing also doesn't happen here. Additionally closing it here will cause the other strategies not to work as they rely on the same page cache.